support-docs/README.md

```diff
@@ -107,7 +109,131 @@ Table below shows the list of properties used to describe pins
+|resistorDerating|Graph to capture resistor derating|Object| |
+
+
+#### 3.9.2	Capacitor Specifications
+The table below gives a description of the properties used to specify a capacitor in a digital datasheet.
+|Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|Value|Capacitance value|Number|Yes|
+|Tolerance|Nominal tolerance of a capacitor|Number| |
+|ratedVoltage|Maximum voltage which may be applied continuously to a capacitance |Number| |
+|dielectric|Dielectric material used in the capacitor|String| |
+|polarized|Describes whether the capacitor is polarized|Boolean| |
+|esr|Equivalent Series resistance of the capacitor|Number| |
+|minTemperature|Minimum temperature under which a capacitor can be expected to reliably operate|Number| |
+|maxTemperature|Maximum temperature under which a capacitor can be expected to reliably operate|Number| |
+|temperatureCoefficient|Change in capacitance when the temperature is changed|Number| |
+|pinSpec|Pins definition of the capacitor|object| |
+|package|Package definition of the capacitor|Object| |
+|complianceList|List of compliances met by the capacitor component|Array| |
+|capacitorDerating|Graph to capture capacitor derating|Object| |
+
+#### 3.9.3	Inductor Specifications
+The table below gives a description of the properties used to specify an inductor in a digital datasheet.
+|Property|Description|Data Type|Required?|
+|:----|:----|:----|:----|
+|Value|Inductance value|Number|Yes|
+|Tolerance|Nominal tolerance of an Inductor|Number| |
+|ratedCurrent|Maximum continuous current the inductor can handle|Number| |
+|saturationCurrent|Current where the inductor enters the magnetic state, and the inductance drops a specified amount |Number| |
+|rmsCurrent|DC current that produces an inductor temperature rise of 40 degrees Celsius|Number| |
+|selfResonantFrequency|Frequency at which the inductor becomes a capacitor|Number| |
+|dcResistance|DC resistance of the inductor|Number| |
+|minTemperature|Minimum temperature under which an inductor can be expected to reliably operate|Number| |
+|maxTemperature|Maximum temperature under which an inductor can be expected to reliably operate|Number| |
+|temperatureCoefficient|Change in inductance when the temperature is changed|Number| |
+|pinSpec|Pins definition of the inductor|object| |
+|package|Package definition of the inductor|Object| |
+|complianceList|List of compliances met by the inductor component|Array| |
+|saturationCurve|Graph to capture saturation curve|Object| |
+|resonantFrequencyCurve|Graph to capture resonant frequency curve|Object| |
+
+#### 3.10	Power Components Specifications
+The The power component specification is included in Github at: https://github.com/edatasheets/edatasheets.github.io
+#### 3.10.1	Buck Converter Specification
+The table below gives a description of the properties used to specify a Buck (step down) switching converter in a digital datasheet.

```

Comment:
technically this is actually for boosts too, right? since we specify "regulatorTopology"

Reply:
Yes. Most of the properties will apply to Boost converters too. The concern is that we didn't look at examples of boost converters datasheets to make sure we were capturing all the main properties. I'm ok changing the title to DC-DC converters specifications, removing references to Bucks and coming back in a later revision to make changes if we need to do so.